management/persistence.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Persistence
+nav_order: 1
+description: How FalkorDB writes data to disk
+parent: Management
+---
+
+Persistence refers to the writing of data to durable storage, such as a solid-state disk (SSD). FalkorDB provides a range of persistence options. These include:
+
+* **RDB** (FalkorDB Database): RDB persistence performs point-in-time snapshots of your dataset at specified intervals.
+* **AOF** (Append Only File): AOF persistence logs every write operation received by the server. These operations can then be replayed again at server startup, reconstructing the original dataset. Commands are logged using the same format as the FalkorDB protocol itself.
+* **No persistence**: You can disable persistence completely. This is sometimes used when caching.
+* **RDB + AOF**: You can also combine both AOF and RDB in the same instance.
+
+To learn more about how to evaluate your FalkorDB persistence strategy, read on.
+
+## RDB advantages
+
+* RDB is a very compact single-file point-in-time representation of your FalkorDB data. RDB files are perfect for backups. For instance you may want to archive your RDB files every hour for the latest 24 hours, and to save an RDB snapshot every day for 30 days. This allows you to easily restore different versions of the data set in case of disasters.
+* RDB is very good for disaster recovery, being a single compact file that can be transferred to far data centers, or onto Amazon S3 (possibly encrypted).
+* RDB maximizes FalkorDB performances since the only work the FalkorDB parent process needs to do in order to persist is forking a child that will do all the rest. The parent process will never perform disk I/O or alike.
+* RDB allows faster restarts with big datasets compared to AOF.
+* On replicas, RDB supports partial resynchronizations after restarts and failovers.
+
+## RDB disadvantages
+
+* RDB is NOT good if you need to minimize the chance of data loss in case FalkorDB stops working (for example after a power outage). You can configure different *save points* where an RDB is produced (for instance after at least five minutes and 100 writes against the data set, you can have multiple save points). However you'll usually create an RDB snapshot every five minutes or more, so in case of FalkorDB stopping working without a correct shutdown for any reason you should be prepared to lose the latest minutes of data.
+* RDB needs to fork() often in order to persist on disk using a child process. fork() can be time consuming if the dataset is big, and may result in FalkorDB stopping serving clients for some milliseconds or even for one second if the dataset is very big and the CPU performance is not great. AOF also needs to fork() but less frequently and you can tune how often you want to rewrite your logs without any trade-off on durability.
```

Comment:
Don't mention AOF here, we're discussing RDB, moreover I don't recall AOF forks.